api/server.ts

```diff
@@ -2,12 +2,39 @@ import express from 'express';
 import bodyParser from 'body-parser';
 import child_process from 'child_process';
 import fs from 'fs';
+import pg from 'pg';
+import formidable from 'express-formidable';
+
 const app = express();
 
 const TEMPLATE = 'dist/__app.html';
 const PORT = 5000;
 const GATOR_DIR = `${__dirname}/../dist/stdlib`;
 
+const pool = new pg.Pool({
```

Comment:
I'm pretty sure this should just work on whatever you run it on (that is, if you do the setup in the readme), but I haven't tested it on machines other than mine,